gui/src/components/Settings/General.tsx

```diff
@@ -42,12 +43,15 @@ export function SettingsGeneral() {
   const onSubmit = useCallback(
     async (data: FieldValues) => {
       reset(data);
+
       await invoke("settings_set", {
         newSettings: data,
       });
-      mutate();
+
+      reloadSettings();
+      reloadTheme();
```

Comment:
isn't this redundant with the store listeners?

Reply:
👍 I was missing a subscribe, updated